src/utils.rs

```diff
@@ -390,6 +390,7 @@ pub(crate) fn new_blockchain(
             retry: wallet_opts.proxy_opts.retries,
             timeout: wallet_opts.electrum_opts.timeout,
             stop_gap: wallet_opts.electrum_opts.stop_gap,
+            validate_domain: false,
```

Comment:
This should be true since that's the default prior to adding the option.
```suggestion
            validate_domain: true,
```

Reply:
ACK.. Missed that thought..